packages/wallets/src/components/ErrorState/ErrorState.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import IcCashierError from '../../public/images/ic-cashier-error.svg';
+import FundsProtection from '../FundsProtection';
+import { GenericPageWithAction } from '../GenericPageWithAction';
+
+type TProps = {
+    error: unknown;
+};
+
+const ErrorState: React.FC<TProps> = ({ error }) => {
+    let code = 'UNKNOWN';
+    let message = 'Sorry an error occured. Please try accessing our cashier page again.';
+
+    if (error && typeof error === 'object' && 'code' in error && typeof error.code === 'string') {
```

Comment:
usage of "typeof" typically is a code smell and usually indicates that something needs to be changed in general structure,
would it make sense to pass "errorMessage: string" as prop instead of error object of "unknown" type?